app/models/curator/institution.rb

```diff
@@ -32,6 +32,21 @@ class Institution < ApplicationRecord
 
     after_update_commit :reindex_associations
 
+    with_options through: :collection_admin_set_objects do
+      has_many :file_sets, source: :file_sets do
+        def exemplaryable
+          where(file_set_type: EXEMPLARYABLE_FILE_SETS)
+        end
+      end
+      has_many :audio_file_sets, source: :audio_file_sets
+      has_many :image_file_sets, source: :image_file_sets
+      has_many :document_file_sets, source: :document_file_sets
+      has_many :ereader_file_sets, source: :ereader_file_sets
+      has_many :metadata_file_sets, source: :metadata_file_sets
+      has_many :text_file_sets, source: :text_file_sets
+      has_many :video_file_sets, source: :video_file_sets
```

Comment:
Could potentially limit this list to the types in `VALID_EXEMPLARY_FILE_SET_TYPES`?